@i18n/ja/translations.yaml

```diff
@@ -17,6 +17,66 @@ topnav.resources.current-status: 現在のステータス
 topnav.resources.explorer: エクスプローラ
 topnav.community.title: コミュニティ
 topnav.community.description: 話題に加わろう。
+
+# index.page.tsx
+home.hero.h1part1: ビジネスのための
+home.hero.h1part2: ブロックチェーン
```

Comment:
I changed the tsx file to use translation IDs here. If we translate the phrases in reverse like this PR had previously, we could encounter surprising and incorrect translation results when the associated phrases were used elsewhere. ("Built for Business" would end up translated as "Blockchain" in katakana, even in other contexts).